Artur_Badalyan/web/html_css/16Jun2021/script.js

```diff
@@ -0,0 +1,130 @@
+registrArray = [];
+
+class Role {
+    firstName = '';
+    lastName = '';
+    age = null;
+
+    readContent() {
+        console.log("This is a content");
+    }
+}
+
+class CreateRole {
```

Comment:
Why do we need this class? If the purpose of it is just initializing users we could do it using a function.